app/src/main/java/com/kickstarter/ui/activities/compose/projectpage/RewardCarouselScreen.kt

```diff
@@ -188,7 +188,7 @@ fun RewardCarouselScreen(
                 ) { reward ->
 
                     val ctaButtonEnabled = when {
-                        RewardUtils.isNoReward(reward) && (project.postCampaignPledgingEnabled() ?: false && project.isInPostCampaignPledgingPhase() ?: false) -> true
+                        RewardUtils.isNoReward(reward) -> true
```

Comment:
Assuming now the Rewards carousel screen is being used un both crowdfund and late pledges a Reward no reward is always available. But @leighdouglas @mtgriego let me know if I'm missing any use case here